src/types/response_types.ts

```diff
@@ -19,3 +39,91 @@ export interface StakingInfo {
 	};
 	validatorSet?: string[] | null;
 }
+
+export interface ISanitizedEvent {
+	method: string;
+	data: EventData;
+}
+
+export interface ISanitizedCall {
+	[key: string]: unknown;
+	method: string;
+	callIndex: Uint8Array | string;
+	args: ISanitizedArgs;
+}
+
+export interface IBlock {
+	number: Compact<BlockNumber>;
+	hash: BlockHash;
+	parentHash: Hash;
+	stateRoot: Hash;
+	extrinsicsRoot: Hash;
+	authorId: AccountId | undefined;
+	logs: ILog[];
+	onInitialize: IOnInitializeOrFinalize;
+	extrinsics: IExtrinsic[];
+	onFinalize: IOnInitializeOrFinalize;
+}
+
+export interface IAccountBalance {
+	at: IAt;
+	nonce: Index;
+	free: Balance;
+	reserved: Balance;
+	miscFrozen: Balance;
+	feeFrozen: Balance;
+	locks: Vec<BalanceLock>;
+}
+
+export interface IAccountStakingSummary {
+	at: IAt;
+	controller: AccountId;
+	rewardDestination: RewardDestination;
+	numSlashingSpans: number;
+	staking: StakingLedger;
+}
+
+export interface IAccountVestingSummary {
+	at: IAt;
+	// eslint-disable-next-line @typescript-eslint/ban-types
+	vesting: Option<VestingInfo> | {};
+}
+
+export interface ITxConstructionMaterial {
+	at: IAt;
+	genesisHash: BlockHash;
+	chainName: string;
+	specName: string;
+	specVersion: U32;
+	txVersion: U32;
+	metadata: string;
+}
+
+interface ILog {
+	type: string;
+	index: number;
+	value: Codec;
+}
+
+interface IOnInitializeOrFinalize {
+	events: ISanitizedEvent[];
+}
+
+interface ISignature {
+	signature: EcdsaSignature | Ed25519Signature | Sr25519Signature;
+	signer: Address;
+}
+
+interface ISanitizedArgs {
+	call?: ISanitizedCall;
+	calls?: ISanitizedCall[];
+	[key: string]: unknown;
+}
+
+interface IExtrinsic {
+	method: string;
+	signature: ISignature | null;
+	nonce: Compact<Index>;
+	args: Codec[];
+	newArgs: ISanitizedArgs;
+}
```

Comment:
`tip, hash, info, events, success, paysFee`?

Reply:
Strange that `tsc` didn't complain here